pkgs/development/python-modules/quart/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, pythonOlder
+, click
+, aiofiles
+, werkzeug
+, jinja2
+, hypercorn
+, itsdangerous
+, blinker
+, hypothesis
+, mock
+, pytestCheckHook
+, pytest-asyncio
+, poetry
+}:
+
+buildPythonPackage rec {
+  pname = "quart";
+  version = "0.17.0";
+  disabled = pythonOlder "3.7";
+  format = "pyproject";
+
+  src = fetchFromGitLab {
+    owner = "pgjones";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-xpMvCOFoSpuit+KnLs3iWwneK++mLgjD7E6vTUUMwaU=";
+  };
+
+  prePatch = ''
+    substituteInPlace pyproject.toml \
+        --replace "--no-cov-on-fail" ""
+  '';
+
+  propagatedBuildInputs = [
+    click
+    aiofiles
+    jinja2
+    hypercorn
+    itsdangerous
+    blinker
+    werkzeug
+    poetry
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    hypothesis
+    mock
+    pytest-asyncio
+  ];
+
+  pythonImportsCheck = [ "quart" ];
+
+  meta = with lib; {
+    homepage = "https://pgjones.gitlab.io/quart/";
+    description = "Python ASGI web microframework";
```

Comment:
This is likely more searchable:

"ASGI web microframework, built as an asyncio alternative to Flask"

Reply:
I have changed it as you suggested. Thanks.